test/topojson/test.ml

```diff
@@ -61,6 +61,12 @@ let pp_position ppf t =
   let lng = Position.lng t in
   Fmt.pf ppf "[%f, %f]" lat lng
 
+let foreign_member ppf v =
```

Comment:
Here you are trying to create a pretty printer for foreign members, this means the signature should look like:

```
val pp_foreign_member : Format.formatter -> (string * Ezjsonm.value) list -> unit
```

So you will probably need something along the lines of 

```ocaml
let pp_foreign_member ppf (v : (string * Ezjsonm.value) list) =
  Fmt.pf ppf "%a" Fmt.(list (pair string pp_ezjsonm)) v
```